.editorconfig

```diff
@@ -0,0 +1,1172 @@
+root = true
```

Comment:
Damn where did you get this from? Not sure what this formatting makes the codes looks like but I'm good without this.

Reply:
This is generated from my IDE following the same formatting I used on the last merged formatting PR 
Code reformatted, reordered entries, optimized imports #946.

Why did you even read it lol it is for the IDE to understand and follow the same formatting for this project.

If this file wasn't included, any changes to the project later on will not be always in the same formatting as the current project formatting. Hence the addition of this file to let the IDE know.